src/components/PlaylistForm.js

```diff
@@ -0,0 +1,57 @@
+import React from "react"
```

Comment:
bug: video is not added to the playlist when a new playlist is created. Please look into it